### DISCUSSION TITLE
JetStream

### DISCUSSION THREAD

**Discussion by DevDanielAlvarez** [tasteui/tasteui#270] [Q&A] [answered]
O CSS e componentes do Tall Stack UI, dá conflito com a lib JetStream? Eu posso usar os dois juntos sem problema ?

**Comment by hitaloramon** [comment, marked as answer]
Não sei informar se daria conflito, pois não testei. Porém no arquivo de config da TallStackUI é possível definir um prefixo.

Por exemplo: prefixando como `ts-`, o uso do `alert` será: `<x-ts-alert />`

**Reply by DevDanielAlvarez** [reply]
Entendi, obrigado pela resposta.
